app/charges/controllers.py

```diff
@@ -112,6 +115,10 @@ def create_charge(user_data):
     charge.committee = committee.id
     charge.status = ChargeStatusType.Unapproved
     charge.priority = ChargePriorityType(user_data["priority"])
+    charge.objectives = user_data["objectives"] if "objectives" in user_data else []
```

Comment:
For all of these you can make it much more clear if you did ```user_data.get('objectives', [])```

Reply:
i did the thing